src/mlstacks/terraform/aws-remote-state/outputs.tf

```diff
@@ -1,3 +1,3 @@
-output "bucket_url" {
-  value = "s3://${aws_s3_bucket.terraform_state.bucket}"
+output "remote_state_bucket_url" {
+  value = module.aws-remote-state.bucket_url
```

Comment:
I don't see `bucket_url` in `main.tf`, only `bucket_name`. Or is `bucket_url` set elsewhere?

Reply:
These come from the respective module output. So https://registry.terraform.io/modules/zenml-io/remote-state/aws/latest?tab=outputs e.g. in the case of AWS.